File: src/Python/KidsWIthTheGreatestNumberOfCandies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        amount = max(candies)
        list = []
        for i in candies:
            if amount <= (i+extraCandies):
                list.append(True)
            else:
                list.append(False)
        return list

File: src/Java/Kids With the GreatestNumberofCandies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> list = new ArrayList<>(); 
        int small = Arrays.stream(candies).max().getAsInt();

        for (int candy : candies) {
            int c = candy + extraCandies;
            if (c >= small)
                list.add(true);
            else
                list.add(false);
        }

        return list;
    }
}